src/io/flutter/FlutterInitializer.java

```diff
@@ -113,6 +114,16 @@ public static void setVerboseLogging(boolean value) {
     properties.setValue(verboseLoggingKey, value);
   }
 
+  public static boolean isMemoryDashboard() {
```

Comment:
I would prefer if we still kept this page under a user preference until we have the following changes done to the page
- toggle to get to the heap snapshot view for reachability information
- filtering/mapping logic added to the page to eliminate the internal VM names and internal VM objects.